src/plugins/Polls/utils.ts

```diff
@@ -0,0 +1,22 @@
+export function dateFormat(fmt: string, date: Date): string {
+    let ret
+    const opt = {
+        'Y+': date.getFullYear().toString(), // year
+        'm+': (date.getMonth() + 1).toString(), // month
+        'd+': date.getDate().toString(), // day
+        'H+': date.getHours().toString(), // hour
+        'M+': date.getMinutes().toString(), // minute
+        'S+': date.getSeconds().toString(), // second
+    }
+    type optKeys = 'Y+' | 'm+' | 'd+' | 'H+' | 'M+' | 'S+'
+    for (let k in opt) {
+        ret = new RegExp('(' + k + ')').exec(fmt)

```

Comment:
Ahh, I'd like to use a library/built-in function than re-inventing wheel like this.